csv2docx/csv2docx.py

```diff
@@ -1,12 +1,47 @@
 import csv
 from mailmerge import MailMerge
-def convert(data, template, delimiter=";"):
-    print ("Getting .docx template and .csv data files ...")
+from pathlib import Path
+
+
+def create_output_folder(path: str) -> Path:
+    """Creates a path to store output data if it does not exists.
+    Args:
+        path: the path from user in any format (relative, absolute, etc.)
+    Returns:
+        A path to store output data.
+    """
+    path = Path(path)
+    if not path.exists():
+        path.mkdir(parents=True, exist_ok=True)
+    return path
+
+
+def create_unique_name(filename: str, path: Path) -> Path:
+    """Checks whether file with same name exists or not
```

Comment:
Can you update this text (`Checks whether ...`) to: `Creates a filename if it does not exist in the specified path.`

Reply:
this would be more clear I guess
"Creates an unique file name in specified path"

Reply:
Yes, that sounds good 👍